src/UiPath.Workflow.Runtime/Statements/FlowNode.cs

```diff
@@ -60,4 +62,11 @@ internal void GetChildActivities(ICollection<Activity> children)
     }
 
     internal abstract void GetConnectedNodes(IList<FlowNode> connections);
+
+    IEnumerable<IFlowNode> IFlowNode.GetConnectedNodes()
+    {
+        var result = new List<FlowNode>();
+        GetConnectedNodes(result);
+        return result;
+    }
```

Comment:
why?

Reply:
There is reflection code in Studio and I would like to take the opportunity to add a proper method.

Reply:
as aa warning, this is internal and I would expect to be able to rename it

Reply:
unfortunately Studio is using it with reflection as there are no contracts available